Slim/Player/StreamingController.pm

```diff
@@ -944,7 +944,8 @@ sub _Continue {
 		_Streamout($self);
 	} elsif (!$bytesReceived || $seekdata) {
 		main::INFOLOG && $log->is_info && $log->info("Restarting stream at offset $bytesReceived");
-		_Stream($self, $event, {song => $song, seekdata => $seekdata, reconnect => 1});
+		# 'reconnect' should not be set as player has *not* set reconnect flag here (see SqueezeBox::reconnect)
+		_Stream($self, $event, {song => $song, seekdata => $seekdata, reconnect => 0});
```

Comment:
So this is the biggest change to before? I'll need to re-read the old discussion around this.

Reply:
Yes, this is the change that allows SB2 to reconnect and restart playing during the "forget_disconnect_timer". The rest is to provide the fix request by the #698

Reply:
I mean, especially having a call to power() when needed so that playback actually restarts. The rest is not just cosmetic :smile: